wiki/Gameplay/Multiplayer/en.md

```diff
@@ -1,434 +1,380 @@
+On hovering over the beatmap, there will be an extra dialogue displaying further information about the beatmap. These are as follows:
 
-For host-side, the host can enable `Free Mods` to allow players to use any mods *except* Speed ([Double Time (DT)](/wiki/Game_modifier/Double_Time)/[Nightcore (NC)](/wiki/Game_modifier/Nightcore) or [Half Time (HT)](/wiki/Game_modifier/Half_Time)) mods.
-Host-selected mods other than the Speed mods will not be enforced upon players.
+| Value | Description |
+| :-: | :-- |
+| `BPM` | Beats per minute |
+| `Length` | Length of the beatmap |
+| `Last Played` | When the beatmap was played the last time |
+| `CS` | Circle Size |
+| `AR` | Approach Rate |
+| `OD` | Overall Difficulty |
+| `HP` | HP Drain Rate |
+| `Stars` | Star Difficulty |
+
+If the player does not have the beatmap, one of these statuses will be shown:
+
+| Beatmap status | Description |
+| :-: | :-- |
+| `Ranked` / `Approved` / `Pending` / `Graveyard` | The [beatmap category](/wiki/Beatmap/Category). Clicking on the beatmap card will open the beatmap page on the website, where it can be [downloaded](/wiki/Beatmap#downloading-beatmaps). |
+| `Not uploaded or not up-to-date` | The beatmap is not available for download. The host should find the link (in the beatmap listing) for the player if the issue persists. |
+| `Cannot update the beatmap` | The [host is probably playing a modified version](https://osu.ppy.sh/community/forums/topics/135726 "peppy's reply on a bug report"). |
 
-#### Team Mode, Win Condition, Tag Colour
+#### Mods
 
-The lower right section will shows how the beatmap will be played in the match (Team Mode) and the Win Condition.
+![](img/Multi_mods_host.jpg "Host mod options")
 
-##### Team Mode
+![](img/Multi_mods_player.jpg "Player (osu!) mods options (with Free Mods enabled)")
 
-Team Mode contains four ways of how a match will be played:-
+The lower left section is the mods section. This section will show the [game mods](/wiki/Game_modifier) used for this match.
 
-| Team Mode | Description |
-| :-- | :-- |
-| `Head to Head` | Compete against each other to reach the top spot of the match leaderboard. |
-| `Team Vs` | Compete against Red/Blue team for match supremacy. |
-| `Tag Coop` \[osu! only, UNRANKED\] | Team up to complete the beatmap, one combo at a time together. |
-| `Tag Team Vs` \[osu! only, UNRANKED\] | Tag Coop, with Red Team and Blue Team match competition. |
+The host can enable `Free Mods` to allow players to freely choose any mod combination, except ones that change gameplay speed ([Double Time (DT)](/wiki/Game_modifier/Double_Time), [Nightcore (NC)](/wiki/Game_modifier/Nightcore) and [Half Time (HT)](/wiki/Game_modifier/Half_Time)).
 
-**Note:** The winner's avatar will be shown at the grade screen for Head to Head Team Mode only.
+#### Team mode {#team-mode-match-setup}
 
-##### Win Condition
+*For more information about team modes, see [Team mode](#team-mode-while-playing).*
```

Comment:
i'm not entirely sure what to do with this tbh, because i can't think of any alternative titles for these headings